rts/lra/test/pom.xml

```diff
@@ -47,11 +45,17 @@
     <lra.coordinator.port>8080</lra.coordinator.port>
     <!-- the property used by NarayanaLRAClient when passed into the container as the JVM parameter -->
     <lra.coordinator.url>http://${lra.coordinator.host}:${lra.coordinator.port}/lra-coordinator</lra.coordinator.url>
+
+    <!-- The following filename is used for starting the WildFly instance that hosts the LRA participant; this file is a copy of the
+     standalone.xml file provided in WildFly (${env.JBOSS_HOME}/standalone/configuration/standalone.xml). Please, see the plugins
+     section of this profile (specifically, maven-antrun-plugin) for further info -->
+    <lra.participant.xml.filename>lra-participant.xml</lra.participant.xml.filename>
     <!-- for test executions where coordinator is started as a separate JVM (e.g. for Quarkus) and the test application
              is deployed at the different JVM; these properties define where the test JVM application is started -->
     <test.application.host>localhost</test.application.host>
     <test.application.port>8180</test.application.port>
-
+    <!-- this property is only used in direct submodules, if used outside this scope it needs to be redined -->
+    <wildfly.trace.logging.cli.script>${project.basedir}/../wildfly-trace-logging.cli</wildfly.trace.logging.cli.script>
```

Comment:
Why have you moved this here? If I understand correctly, this property is only used in the profile `trace.lra`